.github/workflows/cpp.yml

```diff
@@ -352,6 +357,11 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
+      - name: Install CMake 3.25.0
+        shell: bash
+        run: |
+          ci/scripts/install_cmake.sh 3.25.0 /c/cmake
+          echo "c:\\cmake\\bin" >> $GITHUB_PATH
```

Comment:
Do we need this?

It seems that the built-in CMake is enough new:
https://github.com/actions/runner-images/blob/main/images/windows/Windows2019-Readme.md

> CMake 3.31.2